resources/metadata/protecode.yaml

```diff
@@ -0,0 +1,229 @@
+        Triaging is now supported by the Protecode backend and also Piper does consider this information during the analysis of the scan results though product versions are not supported by Protecode. Therefore please make sure that the `fileName` you are providing does either contain a stable version or that it does not contain one at all. By ensuring that you are able to triage CVEs globally on the upload file's name without affecting any other artifacts scanned in the same Protecode group and as such triaged vulnerabilities will be considered during the next scan and will not fail the build anymore.
+spec:
+  inputs:
+    params:
+    - name: protecodeExcludeCVEs
+      type: string
+      description: 'DEPRECATED: Do use triaging within the Protecode UI instead'
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: []
+    - name: protecodeFailOnSevereVulnerabilities
+      type: bool
+      description: Whether to fail the job on severe vulnerabilties or not
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: scanImage
+      aliases:
+      - name: dockerImage
+      type: string
+      description: The reference to the docker image to scan with Protecode
+      mandatory: false
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: container/imageNameTag
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: dockerRegistryUrl
+      type: string
+      description: The reference to the docker registry to scan with Protecode
+      mandatory: false
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: container/registryUrl
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: cleanupMode
+      type: string
+      description: Decides which parts are removed from the Protecode backend after the scan
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: binary
+    - name: filePath
+      type: string
+      description: The path to the file from local workspace to scan with Protecode
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: null
+    - name: includeLayers
+      type: bool
+      description: Flag if the docker layers should be included
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: false
+    - name: addSideBarLink
+      type: bool
+      description: Whether to create a side bar link pointing to the report produced by Protecode or not
+      mandatory: false
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: verbose
```

Comment:
this is a global flag for the command.
Please remove it here.

BTW, as indicated above:
> we probably don't need this since the areas I saw where it was used are handled by the logging automatically.
Maybe I overlooked something though.